packages/lib/src/vis/shared/SelectionTool.tsx

```diff
@@ -77,24 +82,49 @@ function SelectionTool(props: Props) {
       const { sourceEvent, unprojectedPoint } = evt;
       const { target, pointerId } = sourceEvent;
       (target as Element).releasePointerCapture(pointerId);
+      setStartPoint(undefined);
+      setEndPoint(undefined);
 
-      const point = worldToData(boundPointToFOV(unprojectedPoint, camera));
+      if (!modifierKeyPressed) {
+        return;
```

Comment:
Might be more concise to merge this with the next conditional (same in `onPointerMove`).